examples/2d/contributors.rs

```diff
@@ -1,3 +1,32 @@
+//! This example collects a list of all contributors to the bevy source code and renders a bouncing
```

Comment:
Personally, I would prefer if comments explaining a specific system were directly on that system and not all at the top. I think the top level doc should just be a short explanation of what it is. I would start with having the same short description found in the examples README

Reply:
Completely agree here :)